doorway-ui-components/src/actions/Button.scss

```diff
@@ -197,7 +197,7 @@ button {
   --landing-button-active-color: var(--bloom-color-primary);
   --landing-button-text-color: var(--bloom-color-white);
 
-  .button.is-secondary {
+  .doorway-button.is-secondary {
```

Comment:
This fixes the existing bug on prod w the selected button background color on the homepage